yt/visualization/plot_modifications.py

```diff
@@ -55,7 +55,7 @@ class PlotCallback:
     # "figure" this is disregarded.  If "force" is included in the tuple, it
     # will *not* check whether or not the coord_system is in axis or figure,
     # and will only look at the geometries.
-    _supported_geometries: Tuple[str, ...]
+    _supported_geometries: Optional[Tuple[str, ...]] = None
```

Comment:
So the `Optional` is so that it's not required to be defined in subclasses, right?  Not that it isn't required to be defined as an attribute?  This is rather frustrating if this is the case, because if that is correct, it seems like it may be pointless to have *anything* in a base class that is typed as anything other than `Optional`.  Is that the case?